src/main/java/delete_event_use_case/DeleteEventController.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Delete Event Controller class.
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-28, Mon.
+ **/
+
+package delete_event_use_case;
+
+public class DeleteEventController {
```

Comment:
Please add javadoc for all your methods across all classes.

Reply:
Done. Thank you.